.github/workflows/docker.yml

```diff
@@ -667,3 +667,58 @@ jobs:
             failure()
             && github.repository == 'netdata/netdata'
           }}
+
+  trigger-subsequent-workflows:
+    if: github.event_name == 'workflow_dispatch'
+    name: Trigger subsquent workflows for newly added versions
+    needs:
+      - publish-docker-hub
+      - gen-tags
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Trigger Helmchart PR
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.version != 'nightly' && github.repository == 'netdata/netdata'
+        id: trigger-helmchart
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+          repo: netdata/helmchart
+          workflow: Agent Version PR
+          ref: refs/heads/master
+          inputs: '{"agent_version": "${{ needs.gen-tags.outputs.tags }}"}'
```

Comment:
This doesn't work because the workflow in helmchart expects a tag (e.g. v1.45.1) and not 

```
agent-netdata/netdata:v1,netdata/netdata:v1.45,netdata/netdata:v1.45.1,quay.io/netdata/netdata:v1,quay.io/netdata/netdata:v1.45,quay.io/netdata/netdata:v1.45.1,ghcr.io/netdata/netdata:v1,ghcr.io/netdata/netdata:v1.45,ghcr.io/netdata/netdata:v1.45.1
```